dev-modules/babel-plugin-probe-gl/README.md

```diff
@@ -0,0 +1,90 @@
+# babel-plugin-probe-gl
+
+Performs some useful transforms regarding the usage of probe.gl.
+
+## Features
+
+### Remove logs
+
+Remove calls to specified logging methods. This is useful to reduce the logging system's footprint and performance impact in production environment.
+
+The `Log` instance must be called `log` for this feature to work.
+
+#### in
+
+```js
+import log from 'probe.gl';
+
+canvas.addEventListener('click', evt => {
+  log.log(1, `The user clicked the canvas: (${evt.pageX}, ${evt.pageY})`)();
+  try {
+    redraw();
+  } catch (error) {
+    log.error(error.message)();
+  }
+});
+```
+
+#### out
+
+```js
+import log from 'probe.gl';
+
+canvas.addEventListener('click', () => {
+  try {
+    redraw();
+  } catch (error) {
+    log.error(error.message)();
+  }
+});
+```
+
+
+## Installation
+
+```sh
+$ npm install --save-dev babel-plugin-probe-gl
+```
+
+## Usage
+
+### Via `.babelrc` (Recommended)
+
+**.babelrc**
+
+```json
+{
+  "plugins": [["probe-gl", {
+    "removeLogs": ["log", "warn"],
```

Comment:
Would it be hard to make this `removeMethods` so you can remove whatever code you want?

```json
{
  "plugins": [["probe-gl", {
    "removeMethods": {
         "object": "myLogger",
         "methods": ["log", "warn", "myLogging"],
     },
     "patterns": ["*.glsl.js"]
   }]]
}
```